### DISCUSSION TITLE
MAMP for WIndows: PHP version drop down selection is only showing v8.2.14 and v.8.3.1, no way to go lower than 5.6

### DISCUSSION THREAD

**Discussion by ghost** [community/community#146379] [Programming Help]
### Body

Hi, I hope someone here can help me out. I'm currently running MAMP Version: 5.0.6 (7029) x64 binaries release (not PRO) and Windows 10. I have a migration project that I need to run old PHP codes under v5.6, and I wanted to do it first in my local laptop.

When I checked the PHP version drop down list at MAMP control panel, it's only showing v8.2.14 and v.8.3.1. I have `\MAMP\bin\php\php5.6.34` in my laptop, and ran `<?php phpinfo(); ?>` , it showed that active `php.ini` is in `\MAMP\conf\php8.3.1\php.ini`.

I got lost in how to add PHP v5.6 to it (I'm aware that each versions has their separate `php.ini` files, but I'm scared of breaking things at the moment as I have another project that requires the latest PHP version.

Is there anyone here knows how to add an older version of PHP to the MAMP control panel drop down list?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
bump? T_T

**Reply by Singen54** [reply]
اكتب superlinux علي يتيوب
